Fixes:

./pangoview.h:20:9: warning: 'PANGOVIEW_H' is used as a header guard here, followed by #define of a different macro [-Wheader-guard]
#ifndef PANGOVIEW_H
        ^~~~~~~~~~~
./pangoview.h:21:9: note: 'PAGNOVIEW_H' is defined here; did you mean 'PANGOVIEW_H'?
#define PAGNOVIEW_H
        ^~~~~~~~~~~
        PANGOVIEW_H
1 warning generated.

https://github.com/huzheng001/stardict-3/pull/102
--- dict/src/pangoview.h.orig	2012-10-26 07:02:03.000000000 -0500
+++ dict/src/pangoview.h	2019-11-01 06:49:29.000000000 -0500
@@ -18,7 +18,7 @@
  */
 
 #ifndef PANGOVIEW_H
-#define PAGNOVIEW_H
+#define PANGOVIEW_H
 
 #include <list>
 #include <string>
